Modules/Filtering/DisplacementField/include/itkInvertDisplacementFieldImageFilter.hxx

```diff
@@ -109,7 +109,7 @@ InvertDisplacementFieldImageFilter<TInputImage, TOutputImage>::GenerateData()
 
   float oldProgress = 0.0f;
 
-  while (iteration++ < this->m_MaximumNumberOfIterations && this->m_MaxErrorNorm > this->m_MaxErrorToleranceThreshold &&
+  while (iteration++<this->m_MaximumNumberOfIterations && this->m_MaxErrorNorm> this->m_MaxErrorToleranceThreshold &&
```

Comment:
Strange re-spacing of comparison operators

Reply:
Agreed, I'll look into why that occurred.

Reply:
Adding parenthesis around conditionals makes this much easier to read, and fixes the formatting issue.